.contributing/the-process.md

```diff
@@ -12,7 +12,7 @@ Note that the sections are numbered but the items within each section are dot po
 
 -   Create in Drive - a folder called "x. *Section*: Sections Planning (CL)" in "Sections Planning (CL)".  In the folder: 
 
-    -   a Google doc called "0 - Use Cases < *Section* (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases
+    -   a Google doc called "0. Use Cases < *Section* (CL)" that will hold just a list of Use Cases, so we have a short, easy to read and think about, list that is the definitive set of use cases
```

Comment:
changed "-" to "." => keep consistent with naming across doc

Reply:
keen!

Reply:
0 has been exterminated

Reply:
![Alt Text](https://media.makeameme.org/created/feeling-dalek-might.jpg)